README.rst

```diff
@@ -116,6 +119,52 @@ Usage
       latest     Get the latest tag in a changelog.
       tag        Create an annotated git tag based on a changelog entry.
       update     Updates the version and date of the most recent changelog entry.
+      create     Creates a new CHANGELOG.md file with a standard header.
+      entry      Add a quick entry to the current version.
+      next       Create a new unreleased version in changelog file.
+
+next
+----
+
+Create a new unreleased version in changelog file.
+
+::
+
+   Usage: chag next
+
+   Creates a new unreleased version inside of the changelog file.
+
+   Options:
+     --help     Displays this message.
+
+entry
```

Comment:
I created [this issue](https://github.com/dpsenner/chag/issues/2). I would like anyone invite to join the conversation.